data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,646 @@ mission "Remnant: Learn Sign Follow Up"
+			`	The guard escorts you out of the cell, which was inside the security office of the Institute near the lobby. You get some puzzled looks from those in the lobby as you're escorted by the security guard in cuffs. After bringing you outside, the guard unlocks the cuffs. "Don't cause any more trouble, now," he says.`
+				decline
+			
+			label escorted
+			`	The security guards escort you out of Ivan's office, and then out of the building. Looks like the Remnant won't be getting what they were hoping for.`
+				decline
+				
+			label out
+			`	You back out of Ivan's office with him looking at you the entire time. Looks like the Remnant won't be getting what they were hoping for.`
+				decline
+			
+			label leave
+			`	Ivan smiles, but it looks forced. "Come back any time you want. You're free to roam the Institute whenver you desire. Most of the building is closed to the public, but I can always mark you as my guest whenever you're interested in looking around."`
+				decline
+			
+			label cubes
+			`	Ivan laughs as he reclines his chair. "Screw that. You should see what I've been able to recover from those sensor cubes you placed.`
+				goto ramble
+			
+			label surveillance
+			choice
+				`	"I was wondering what came of those sensor cubes I placed."`
+				`	"Could I get a copy of what you've found from those sensor cubes?"`
+					goto "surveillance copy"
+			`	Ivan laughs as he reclines his chair. "Oh, it's absolutely fascinating what I've been able to find.`
+			label ramble
+			`	"With a cube in each of the Remnant systems, I was able to track all their fleet movements, and I've discovered some interesting things. For one, ships either make trips between the three systems, or they leave the cluster entirely for significant amounts of time, often at least a month at a time. There weren't very many ships or fleets that took short trips out of the cluster, say only one or two weeks in length. Seems like they're sending people on rather lengthy missions.`
+			`	"Aside from fleet movements, the cubes are able to make low quality scans of any ships in the system. While those individual scans are less than what an outfit scan provides, given enough scans you're able to get a clearer picture. Now there were specific ship models that would suddenly disappear from the sensors without a trace. I knew they couldn't have been jumping out of the system because jumping creates a very distinct signature on the sensors. On top of ships disappearing on the sensors, there were also jump signatures with no disceranble source. My only conclusion then is that some of these ships are capable of cloaking themselves.`
+			`	"I actually lost connection to the cubes a few weeks ago now, but I've gathered enough information already."`
```

Comment:
I'm really iffy about how much data they are getting here, and especially how *long* they are getting data. 

If it was less detailed than maybe I could spin it with future missions that the Remnant were "feeding" it data after they found them, but there's far too much reasonably accurate info here for that. 

Given the mix of:
- Remnant security
- Power of Remnant scanners
- In-game missions already make it clear that they are scanning the player's ship whenever they come in.

They should know at the very least what First is bringing to the planet, how big it is, and whether they took it with them them when they left immediately. Then there is First's definite traceability, and the ease with which Remnant locate them all the time, which suggests either they are tracking first, or have someone following them. 

In any case, those sensor cubes should be found within a day, probably more likely within hours. As such, a more likely series of events is that the Remnant locate the devices within hours, so the devices get a few hours of scans; beyond which all the info they collect is being carefully selected and fed by the Remnant. 

As such, I have two suggestions for modifying this section.
1) Have Ivan mention that they are getting an unusual amount of interference, perhaps the ion storms?
2) Remove the part about jump signatures with no discernible source, and rephrase the section to suspect stealth technology of some kind. 

The rest seems reasonable, and  could be interpreted a few different ways; and that ambiguity is good.

Reply:
```suggestion
			`	"With a cube in each of the Remnant systems, I was able to track all their fleet movements, and I've discovered some interesting things. For one, ships either make trips between the three systems, or they leave the cluster entirely for significant amounts of time, often at least a month at a time. There weren't very many ships or fleets that took short trips out of the cluster, say only one or two weeks in length. Seems like they're sending people on rather lengthy missions.`
			`	"Aside from fleet movements, the cubes are able to make low quality scans of any ships in the system. There seems to be some sort of interference, so individual scans are less than what an outfit scan provides, but given enough scans you're able to get a clearer picture. Now there were specific ship models that would suddenly disappear from the sensors without a trace, usually just as they passed through the shadow of a moon or asteroid, never to reappear. I knew they couldn't have been jumping out of the system because jumping creates a very distinct signature on the sensors. My only conclusion then is that some of these ships are equipped with stealth technology.`
			`	"I actually lost connection to the cubes a few weeks ago now, but I've gathered enough information already."`
```

Reply:
Removed mention of the cloaking/stealth altogether.